drawee/src/main/java/com/facebook/drawee/view/SimpleDraweeView.java

```diff
@@ -48,38 +50,55 @@ public static void shutDown() {
 
   public SimpleDraweeView(Context context, GenericDraweeHierarchy hierarchy) {
     super(context, hierarchy);
-    init();
+    init(context, null);
   }
 
   public SimpleDraweeView(Context context) {
     super(context);
-    init();
+    init(context, null);
   }
 
   public SimpleDraweeView(Context context, AttributeSet attrs) {
     super(context, attrs);
-    init();
+    init(context, attrs);
   }
 
   public SimpleDraweeView(Context context, AttributeSet attrs, int defStyle) {
     super(context, attrs, defStyle);
-    init();
+    init(context, attrs);
   }
 
   @TargetApi(Build.VERSION_CODES.LOLLIPOP)
   public SimpleDraweeView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
     super(context, attrs, defStyleAttr, defStyleRes);
-    init();
+    init(context, attrs);
   }
 
-  private void init() {
+  private void init(Context context, @Nullable AttributeSet attrs) {
     if (isInEditMode()) {
       return;
     }
     Preconditions.checkNotNull(
         sDraweeControllerBuilderSupplier,
         "SimpleDraweeView was not initialized!");
     mSimpleDraweeControllerBuilder = sDraweeControllerBuilderSupplier.get();
+
+    if (attrs != null) {
+      TypedArray gdhAttrs = context.obtainStyledAttributes(
+          attrs,
+          R.styleable.SimpleDraweeView);
+      try {
+        final int indexCount = gdhAttrs.getIndexCount();
+        for (int i = 0; i < indexCount; i++) {
```

Comment:
Since here we handle only a single attribute, can you get `SimpleDraweeView_imageUri` directly instead of looping over all of them?